cmd/zfs/zfs_main.c

```diff
@@ -7081,6 +7086,41 @@ get_stats(nvlist_t *outnvl)
 	return (jobj);
 }
 
+static int
+zfs_do_list_snap(int argc, char **argv)
+{
+	nvlist_t *outnvl = NULL;
+	nvpair_t *elem = NULL;
+	int error = 0;
+	char *str_val;
+
+	if (argc != 2) {
+		fprintf(stderr, "got list snap command %d\n", argc);
+		usage(B_FALSE);
+	}
+
+	if ((error = lzc_list_snap(argv[1], NULL, &outnvl)) != 0) {
+		fprintf(stderr, "failed list snap command for %s with err %d\n",
```

Comment:
nit: be consistent with naming, should be "listsnap command"

Reply:
will take care in next commit